src/transforms/babel/config.js

```diff
@@ -44,7 +44,12 @@ async function getBabelConfig(asset) {
     // Filter out plugins that are already applied by @babel/preset-env
     if (Array.isArray(babelrc.config.plugins)) {
       babelrc.config.plugins = babelrc.config.plugins.filter(plugin => {
-        return !ENV_PLUGINS[getPluginName(plugin)];
+        // we should test only packages starts with '@babel/' prefix here
+        const pluginName = getPluginName(plugin);
+        if (pluginName.indexOf('@babel/') === 0) {
```

Comment:
I don't think we want this if statement here. We also want to detect duplicates with the babel6 versions, right?

Reply:
Good point, I agree. But only way I see here is add `babel-preset-env` as dependency for importing different set of plugins for babel 6